wazo_call_logd/generator.py

```diff
@@ -22,6 +23,114 @@
 )
 
 
+class _ParticipantsProcessor:
+    def __init__(self, confd_client: ConfdClient):
+        self.confd: ConfdClient = confd_client
+        self.confd_participants: dict[str, ParticipantInfo] = {}
+
+    def __call__(self, call_log: RawCallLog) -> RawCallLog:
```

Comment:
Not a fan of the `__call__` method. Suggestion: `process(...)`

Reply:
In my mind, if a class exists only to provide encapsulation of a single method, and so is equivalent to(could be replaced by) a function closure(some environment + some parameterized code using that environment), it deserves a callable interface.